### DISCUSSION TITLE
Feature Request - Add Spatial-Configuration-Net (SCN) for Landmark Localization in 2D/3D images

### DISCUSSION THREAD

**Discussion by gofmans** [Project-MONAI/MONAI#6180] [Ideas]
Hi,

It would be great to have an additional network architecture in MONAI that can localize landmarks in 2D/3D images.

More specifically, I encountered spatial-configuration-net that uses heatmaps regression for this task, as described at:
https://arxiv.org/pdf/1908.00748v1.pdf
![image](https://user-images.githubusercontent.com/26842519/226290345-9c5ea79b-32b1-4b68-a20a-03002fdd7e6a.png)

Moreover, here is an example of using this network for localizing vertebra in 3D volumes: 
https://cpb-ap-se2.wpmucdn.com/blogs.auckland.ac.nz/dist/1/670/files/2020/06/2020PayerVISAPP.pdf
![image](https://user-images.githubusercontent.com/26842519/226290831-96204433-f7c0-439f-b860-53bf9eb524bb.png)

Is there any existing model under monai.networks.nets that is dedicated for the task of localizing landmarks? If not, having a dedicated model would be great.

Keep up the awesome work!

**Comment by wyli** [comment]
please see
- https://github.com/Project-MONAI/model-zoo/tree/dev/models/valve_landmarks
- https://github.com/Project-MONAI/MONAILabel/blob/fullCTSegmentation/sample-apps/radiology/lib/configs/localization_vertebra.py (cc @diazandr3s might have more examples on this one)

**Reply by diazandr3s** [reply]
Thanks for highlighting this, @gofmans.

As shown here (https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#multistage-vertebra-segmentation), our reference vertebra segmentation model in MONAI Label doesn't use regression. Instead, Stage 2: Vertebra Localization is a segmentation task. Once we get the vertebra segmented, we compute the centroid.

It is true that I didn't find specifically the SpatialConfiguration-Net (SCN) in MONAI, but MONAI has all the tools for you to build one: https://github.com/Project-MONAI/MONAI/tree/dev/monai/networks/layers

@rijobro may also want to comment on this :)

**Comment by GauthierDot** [comment]
Hi everyone,

Thank you for your awesome work on MONAI. I'm also greatly interested in landmarks localization on 3D images (especially using SCN), and would like to know if there were any news on that matter from your side :) ?

Best,
Gauthier
